<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="EditWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="EditLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="XLabel">
          <property name="text">
           <string>X Label</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TitleLabel">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="YLabel">
          <property name="text">
           <string>Y Label</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="PlotTypeCombo">
          <item>
           <property name="text">
            <string>Line Plot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scatter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Horizontal Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stacked Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Horizontal Stacked Bar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stacked</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Timeline</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3D Scatter</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3D Line</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="ZLabelLine"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="ZLabel">
          <property name="text">
           <string>Z Label</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="XLabelLine"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="YLabelLine"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="TitleLine"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="PlotTypeLabel">
          <property name="text">
           <string>Plot Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ClassificationLabel">
          <property name="text">
           <string>Classification</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ClassificationLine"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="4" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Code By Uniques</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;Line Plot Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Y Axis Data</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>X Axis Data</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="LineXAxisHeader"/>
              </item>
              <item row="3" column="1">
               <widget class="QListWidget" name="LineYAxisHeaders">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="LineSearchBar"/>
              </item>
              <item row="4" column="1">
               <widget class="QTableWidget" name="LineCodeByTable">
                <column>
                 <property name="text">
                  <string>Y Axis Header</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Source Header</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Modification Type</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QListWidget" name="ScatterYAxisHeaders">
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="ScatterXAxisHeader"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Y Axis Data</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Code By Uniques</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>X Axis Data</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="ScatterSearchBar"/>
              </item>
              <item row="4" column="1">
               <widget class="QTableWidget" name="ScatterCodeByTable">
                <column>
                 <property name="text">
                  <string>Y Axis Header</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Source Header</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Modification Type</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Colormap</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Colorbar Label</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;Scatter Plot Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>PlotTypeCombo</tabstop>
  <tabstop>ClassificationLine</tabstop>
  <tabstop>TitleLine</tabstop>
  <tabstop>XLabelLine</tabstop>
  <tabstop>YLabelLine</tabstop>
  <tabstop>ZLabelLine</tabstop>
  <tabstop>LineXAxisHeader</tabstop>
  <tabstop>LineYAxisHeaders</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>PlotTypeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClassificationLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TitleLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XLabelLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YLabelLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZLabelLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Plot()</slot>
 </slots>
</ui>
